l5kit/l5kit/environment/__init__.py

```diff
@@ -0,0 +1,9 @@
+import gym
+
+
+# Register L5 Env
+gym.envs.register(
+    id='L5-CLE-v0',
+    entry_point="l5kit.environment.envs.l5_env:L5Env",
+    kwargs={'env_config_path': '../../../examples/RL/config.yaml'},
```

Comment:
Could you suggest a way to handle this error? This will also solve the previous comment regarding "having a default for `env_config_path` to be the `config.yaml `